data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/TableMetadataUtils.java

```diff
@@ -0,0 +1,151 @@
+package com.scalar.db.dataloader.core.util;
+
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.io.DataType;
+import com.scalar.db.transaction.consensuscommit.Attribute;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/** Utils for ScalarDB table metadata */
+public final class TableMetadataUtils {
+
+  /** Format for the lookup key for a table in a namespace */
+  public static final String TABLE_LOOKUP_KEY_FORMAT = "%s.%s";
+
+  /** Private constructor to prevent instantiation */
+  private TableMetadataUtils() {}
+
+  /**
+   * Check if the field is a ScalarDB transaction metadata column or not
+   *
+   * @param columnName Table column name
+   * @param metadataColumns Fixed list of metadata columns
+   * @param columnNames List of all column names in a table
+   * @return The field is metadata or not
+   */
+  public static boolean isMetadataColumn(
```

Comment:
We might be able to use [ConsensusCommitUtils](https://github.com/scalar-labs/scalardb/blob/03f7321282d0e335f333197af2a0a59b07f12c21/core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java). Could you please check it?